src/mlpack/methods/ann/ffn_impl.hpp

```diff
@@ -642,6 +642,11 @@ FFN<OutputLayerType, InitializationRuleType, CustomLayers...>::FFN(
     this->network.push_back(boost::apply_visitor(copyVisitor,
         network.network[i]));
   }
+
+  for (const auto& net:this->network)
```

Comment:
Do you think we could make this a part of the previous loop?  Let me know if I overlooked something.

Reply:
Yeah it is better with single loop. I changes this.